code-of-conduct.md

```diff
@@ -1,43 +1,64 @@
-# OpenContainers Code of Conduct
+# OpenContainers Code of Conduct v1.0
```

Comment:
I recommend just using the [template][1] with the email replacement.  Why adapt the title?  I haven't done a detailed survey, but the first project I checked [stuck with the stock title][2].

[1]: contributor-covenant.org/version/1/4/code_of_conduct.md
[2]: https://github.com/atom/atom/blob/v1.8.0/CODE_OF_CONDUCT.md#contributor-code-of-conduct